packages/amazonq/src/lsp/chat/messages.ts

```diff
@@ -184,6 +206,89 @@ export function registerMessageListeners(
                 break
         }
     }, undefined)
+
+    const registerHandlerWithResponseRouter = (command: string) => {
```

Comment:
nit: Instead of defining a function as a const before usage I like to define it as a nested function at the end of the root function. I do it since it shifts the lower level implementation details all to the end of the function, for IF I want to read it.

I know you already are familiar with this, but thought I'd mention as I personally find it is the better practice to follow